examples/main/utils/color-picker.js

```diff
@@ -0,0 +1,57 @@
+/* global document */
+import React, {PureComponent} from 'react';
+
+function pad2(str) {
```

Comment:
While the functions in this file are somewhat self explanatory, it would be good to have at least one line of comment for each function describing what they do.